examples/routing/routing-project/src/components/modules/Router/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Island } from '@hubspot/cms-components';
+import AppIsland from '../../islands/App?island';
+
+export const Component = () => {
+  return (
+    <div>
```

Comment:
Just a curiosity, is this DIV needed here?

Reply:
Nope! Just a little leftover cruft 😅